examples/seq2seq/README.md

```diff
@@ -15,55 +15,7 @@ For `bertabs` instructions, see [`bertabs/README.md`](bertabs/README.md).
 
 ## Datasets
 
-#### XSUM:
-```bash
-cd examples/seq2seq
-wget https://cdn-datasets.huggingface.co/summarization/xsum.tar.gz
-tar -xzvf xsum.tar.gz
-export XSUM_DIR=${PWD}/xsum
-```
-this should make a directory called `xsum/` with files like `test.source`.
-To use your own data, copy that files format. Each article to be summarized is on its own line.
-
-#### CNN/DailyMail
-```bash
-cd examples/seq2seq
-wget https://cdn-datasets.huggingface.co/summarization/cnn_dm_v2.tgz
-tar -xzvf cnn_dm_v2.tgz  # empty lines removed
-mv cnn_cln cnn_dm
-export CNN_DIR=${PWD}/cnn_dm
-```
-this should make a directory called `cnn_dm/` with 6 files.
-
-#### WMT16 English-Romanian Translation Data:
-download with this command:
-```bash
-wget https://cdn-datasets.huggingface.co/translation/wmt_en_ro.tar.gz
-tar -xzvf wmt_en_ro.tar.gz
-export ENRO_DIR=${PWD}/wmt_en_ro
-```
-this should make a directory called `wmt_en_ro/` with 6 files.
-
-#### WMT English-German:
-```bash
-wget https://cdn-datasets.huggingface.co/translation/wmt_en_de.tgz
-tar -xzvf wmt_en_de.tgz
-export DATA_DIR=${PWD}/wmt_en_de
-```
-
-#### Private Data
-
-If you are using your own data, it must be formatted as one directory with 6 files:
-```
-train.source
-train.target
-val.source
-val.target
-test.source
-test.target
-```
-The `.source` files are the input, the `.target` files are the desired output.
-
+See [Getting evaluation data](./eval_datasets.md).
```

Comment:
(1) Never mind my copy paste comment about the original contents of this Section. They should stay here.  This hyperlink should just say, roughly 
> To replicate FSMT or Pegasus, see `.eval_datasets.md`.

(2) Consider separating eval_fsmt.md and replicate_pegasus.md, they seem like very different use cases.

(3) Alternatively, you could try to think about how this whole system should work and try to make documentation easy to navigate for different use cases -- (evaluating a model), (training a model), replicating a paper. If you go this route, keep all the stuff about using your own data high in the page, as this is a large fraction of the training use cases.

Reply:
Please clarify number 1? you mean to leave everything as is, right? How about I just add pegasus and fsmt notes to README.md and we are done?

I'd be happy to write better docs once I work on such project. I can't do it in void.

Reply:
> How about I just add pegasus and fsmt notes to README.md and we are done?

Yes. But then exit the void :), and make sure that when you read the document as a whole, it coheres.

Reply:
Slack if doesn't make sense, I'm just trying to (a) make your doc discoverable without (b) making existing stuff less clear.

Reply:
I added the information in a very succinct way - so it shouldn't interfere with the current doc.